cairo-contracts/packages/clients/src/cometbft/consensus_state.cairo

```diff
@@ -13,6 +13,10 @@ pub struct CometConsensusState {
 
 #[generate_trait]
 pub impl CometConsensusStateImpl of CometConsensusStateTrait {
+    fn zero() -> CometConsensusState {
+        CometConsensusState { timestamp: TimestampZero::zero(), root: StateRootZero::zero() }
+    }
```

Comment:
you should rather implement `core::num::traits::Zero` for `CometConsensusState`.

Reply:
Done in 464b816bf4b4231e9e08f27ba015a1b8455e470d